src/librustc_typeck/check/dropck.rs

```diff
@@ -83,10 +87,7 @@ fn ensure_drop_params_and_item_params_correspond<'tcx>(
         let fresh_impl_self_ty = drop_impl_ty.subst(tcx, fresh_impl_substs);
 
         let cause = &ObligationCause::misc(drop_impl_span, drop_impl_hir_id);
-        match infcx
```

Comment:
please try to avoid making pure-formatting changes in the same commit that has effectual semantic changes.

(If the reformatting is being injected by e.g. running `rustfmt`, then try to do those runs in a separate commit, please.)

Reply:
(the reason I'm asking for it to be in a separate commit is that it eases review: It allows the reviewer to just quickly skim the commits that are formatting fixes, while diving into the commits that have the real meat!)

Reply:
Btw, ticking the "hide whitespace changes" checkbox usually helps a bit with this when reviewing.